llvm/test/MC/X86/msrimm-64-att.s

```diff
@@ -0,0 +1,10 @@
+// RUN: llvm-mc -triple x86_64 --show-encoding %s | FileCheck %s
```

Comment:
Add `# RUN: not llvm-mc -triple i386 -show-encoding %s 2>&1 | FileCheck %s --check-prefix=ERROR` ?

Reply:
86711f3